### PR TITLE
forecast and accuracy generics

### PR DESCRIPTION
**Pull request by config-i1**
This is a pull request to fix the issue #63

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: generics
 Title: Common S3 Generics not Provided by Base R Methods Related
     to Model Fitting
-Version: 0.1.1.9000
+Version: 0.1.2.9000
```
Comment (mitchelloharawild) [inline DESCRIPTION:4]: Probably doesn't need the version bump included in the PR.

**Review by mitchelloharawild — APPROVED** [review]
Looks good to me, thanks for the PR. Other than the bump to a future dev version, I think the PR is ready and would be useful to resolve generic conflicts between the `{forecast}`, `{fabletools}` and `{greybox}` packages.

**Comment by hadley** [comment]
@config-i1 do you want to make this change?

**Comment by config-i1** [comment]
@hadley I'm okay with the changes done by @mitchelloharawild, if that's what you mean :)

**Comment by config-i1** [comment]
Ah! Sorry. I haven't spotted that. Done.

**Comment by hadley** [comment]
Thanks! I'll get out a release in the near future.

**Pull request merged**